component/web/security/src/main/java/org/exoplatform/web/login/RememberMeFilter.java

```diff
@@ -48,14 +52,28 @@ public class RememberMeFilter extends AbstractFilter {
     //value of this field need equals with: org.gatein.security.oauth.common.OAuthConstants.ATTRIBUTE_AUTHENTICATED_PORTAL_USER_FOR_JAAS
     public static final String ATTRIBUTE_AUTHENTICATED_PORTAL_USER_FOR_JAAS = "_authenticatedPortalUserForJaas";
 
+    public List<String>      ignoredPaths                                 = null;
+
+    @Override
+    protected void afterInit(FilterConfig config) throws ServletException {
+      String ignoredPathsParameter = config.getInitParameter("ignoredPaths");
+      if (StringUtils.isBlank(ignoredPathsParameter)) {
```

Comment:
Shoud we use 
```suggestion
      if (ignoredPathsParameter==null || ignoredPathsParameter.isBlank()) {
```

to prevent using StringUtils library ?

Reply:
why preventing using commons library ?

Reply:
To use standard java way. as java provide the possibility to check strings, is it correct to use external lib for that ? 

It is only a reflexion, not an objection, I should have put it in comment instead of Change Request

Reply:
Ok, let's discuss about this basic and general concern. In fact, what is made here is the purpose of third party libraries making the code more lisible and centralize the same concern in a re-usable way, even Spring has defined their own StringUtils class.